src/main/java/betterquesting/handlers/EventHandler.java

```diff
@@ -724,6 +724,12 @@ public void onEntityCreated(EntityJoinWorldEvent event) {
         PlayerContainerListener.refreshListener((EntityPlayer) event.getEntity());
     }
 
+    @SubscribeEvent
+    public static void onPlayerLoggedOut(PlayerEvent.PlayerLoggedOutEvent event) {
```

Comment:
Event listener cannot be static, we're registering an instance of `EventHandler` to the event bus.

Reply:
This is still blocking the PR